src/components/Graph/Graph.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Node from './Node';
+import Edge from './Edge';
+
+const Graph = ({ nodes, edges }) => {
+  const renderNodes = nodes.map((node, index) => (
+    <Node
+      key={index}
+      x={node.x}
+      y={node.y}
+    />
+  ));
+
+  const renderEdges = edges.map((edge, index) => (
+    <Edge
+      key={index}
+      x1={edge.source.x}
+      y1={edge.source.y}
+      x2={edge.target.x}
+      y2={edge.target.y}
+    />
+  ));
+
+  return (
+    <div
+      style={
+        {
+          position: 'relative',
+          width: '500px',
+          height: '500px',
+          border: '1px solid black'
+        }
+      }>
+      {renderNodes}
```

Comment:
Avoid inline styles, use CSS (I recommend css modules over global css)

Reply:
That it temporary until we figure out which library we use